images/cluster-node-tuning-operator/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM centos:7
+
+RUN yum install -y golang make
+
+ENV GOPATH /go
+RUN mkdir $GOPATH
+
+COPY . $GOPATH/src/github.com/openshift/cluster-node-tuning-operator
+
+RUN cd $GOPATH/src/github.com/openshift/cluster-node-tuning-operator \
+    && make build \
+    && cp $GOPATH/src/github.com/openshift/cluster-node-tuning-operator/cluster-node-tuning-operator /usr/bin/
+
+RUN yum remove -y golang make
+
+ENTRYPOINT ["/usr/bin/cluster-node-tuning-operator"]
+
+USER 1001
```

Comment:
whats this?

Reply:
This will be cleaned up in future PRs.  Might even use a completely different Dockerfile.